### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by jsoref**
Please feel free to squash/reorder/split commits...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,31 +237,31 @@ The following are valid user configuration options.
...
-IpForwarding      | target_instance_ips  | `false` disables internal IP address load balancing.
-MetadataScripts   | run_dir              | String base directory where metadata scripts are executed.
+Accounts          | useradd\_cmd          | Command string to create a new user.
+Accounts          | userdel\_cmd          | Command string to delete a user.
+Accounts          | usermod\_cmd          | Command string to modify a user's groups.
+Accounts          | groupadd\_cmd         | Command string to create a new group.
+Daemons           | accounts\_daemon      | `false` disables the accounts daemon.
+Daemons           | clock\_skew\_daemon    | `false` disables the clock skew daemon.
+Daemons           | ip\_forwarding_daemon | `false` disables the IP forwarding daemon.
+InstanceSetup     | optimize\_local_ssd   | `false` prevents optimizing for local SSD.
```
Comment (illfelder) [inline README.md]: The second "_" is not escaped in the previous two config options.

I'm not sure what the escaping changes - both appear to render the same.

Comment (jsoref) [inline README.md]: As long as there aren't two unescaped _'s, markdown doesn't do anything silly, I meant to escape all of them to be consistent. Thanks.

Comment (jsoref) [inline README.md]: Fwiw, this is how some markdown clients render the table (without the patch):
image

Comment (jsoref) [inline README.md]: https://www.question-defense.com/2010/02/12/include-underscores-while-using-markdown-without-them-being-bold-or-italic-indicators-for-text

**Comment by jsoref** [comment]
@illfelder : done...

**Pull request merged**